### DISCUSSION TITLE
Add per-game controller emulation choice

### DISCUSSION THREAD

**Discussion by Hollyrious** [LizardByte/community#264] [Sunshine]
It'd be amazing if we could have per-game controller choice. For example, I use a Dualsense controller on PC, with most of my games modded to show PS glyphs - however, some I have DS4Windows set to instead emulate a Dualshock controller as it's natively supported. My AYN Odin has custom PS buttons, as such any game using a xinput with mods works great, but anything requiring a native dualshock will still show Xbox glyphs.

Being able to choose a controller override per-game would completely solve this.

**Comment by ReenigneArcher** [comment]
You submitted this first, but there is a duplicate that has more activity: https://github.com/orgs/LizardByte/discussions/302
